rails-commenting.rb

```diff
@@ -55,14 +62,17 @@ def destroy
       redirect_to blog_posts_path
     else
       # ---8)
+      # if the destroy fails for whatever reason, return to the url of the blog post identified in @post
       redirect_to blog_post_path(@post)
     end
   end
 
-  # ---9)
+  # ---9)  
+  # these are methods hidden from the user, but accessible to the developer
```

Comment:
http://ruby-for-beginners.rubymonstas.org/advanced/private_methods.html
In case, you needed more info about the private method